modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'evss/ppiu/service'
+
+module Mobile
+  module V0
+    class PaymentInformationController < ApplicationController
+      before_action { authorize :evss, :access? }
+      before_action { authorize :ppiu, :access? }
+      before_action :validate_pay_info, only: :update
+      before_action(only: :update) { authorize(:ppiu, :access_update?) }
+
+      def index
+        payment_information = service.get_payment_information.responses[0]
+        render json: Mobile::V0::PaymentInformationSerializer.new(@current_user.id,
+                                                                  payment_information.payment_account,
+                                                                  payment_information.control_information)
+      end
+
+      def update
+        payment_information = service.update_payment_information(pay_info).responses[0]
+        send_confirmation_email
+        render json: Mobile::V0::PaymentInformationSerializer.new(@current_user.id,
+                                                                  payment_information.payment_account,
+                                                                  payment_information.control_information)
+      end
+
+      private
+
+      def service
+        @service ||= EVSS::PPIU::Service.new(@current_user)
+      end
+
+      def ppiu_params
+        params.permit(
+          :account_type,
+          :financial_institution_name,
+          :account_number,
+          :financial_institution_routing_number
```

Comment:
It looks like the `filter_parameters` lambda is a whitelist, but still may be good to confirm none of these get into logs.

Reply:
Did some digging and didn't find anything that looked to be logging those values